### PR TITLE
Rspec for User

### PR DESCRIPTION
**Pull request by nky4593**
Screenshot from 2019-07-29 10-23-59
Screenshot from 2019-07-29 10-25-25

Link Redmine : https://edu-redmine.sun-asterisk.vn/issues/15723

### CODE REVIEW

**Comment by nky4593** [comment]
ready

File: spec/examples.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+example_id                      | status | run_time        |
```
Comment (hungct-1121) [inline spec/examples.txt]: cái này ko đưa lên anh ơi

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  describe "Create" do
+    it { is_expected.to be_valid }
+  end
+
+  it "has a valid" do
+    expect(subject).to be_valid
+  end
+
+  it "is invalid  without a emial" do
```
Comment (hungct-1121) [inline spec/models/user_spec.rb]: email :v

File: spec/examples.txt (hunk 1 above)
Comment (nky4593) [reply spec/examples.txt]: ok chú

File: spec/models/user_spec.rb (hunk 2 above)
Comment (nky4593) [reply spec/models/user_spec.rb]: anh sửa r nhé, xấu hổ quá :v

**Comment by nky4593** [comment]
ready

**Pull request merged**